docs/source/usersguide/settings.rst

```diff
@@ -0,0 +1,223 @@
+:class:`openmc.stats.Box`. To independently specify distributions in the x, y,
+and z coordinates, you can use :class:`openmc.stats.CartesianIndependent`.
+
+The angular distribution can be set equal to a sub-class of
+:class:`openmc.stats.UnitSphere` such as :class:`openmc.stats.Isotropic`,
+:class:`openmc.stats.Monodirectional`, or
+:class:`openmc.stats.PolarAzimuthal`. By default, if no angular distribution is
+specified, an isotropic angular distribution is used.
+
+The energy distribution can be set equal to any univariate probability
+distribution. This could be a probability mass function
+(:class:`openmc.stats.Discrete`), a Watt fission spectrum
+(:class:`openmc.stats.Watt`), or a tabular distribution
+(:class:`openmc.stats.Tabular`). By default, if no energy distribution is
+specified, a Watt fission spectrum with :math:`a` = 0.988 MeV and :math:`b` =
+2.249 MeV :sup:`-1` is used.
+
+As an example, to create an isotropic, 10 MeV monoenergetic source uniformly
+distributed over a cube centered at the origin with an edge length of 10 cm, one
+would run::
+
+  source = openmc.Source()
+  source.space = openmc.stats.Box((-5, -5, -5), (5, 5, 5))
+  source.angle = openmc.stats.Isotropic()
+  source.energy = openmc.stats.Discrete([10.0e6], [1.0])
+  settings.source = source
+
+The :class:`openmc.Source` class also has a :attr:`Source.strength` attribute
+that indicates the relative strength of a source distribution if multiple are
+used. For example, to create two sources, one that should be sampled 70% of the
+time and another that should be sampled 30% of the time::
+
+  src1 = openmc.Source()
+  src1.strength = 0.7
+  ...
+
+  src2 = openmc.Source()
+  src2.strength = 0.3
+  ...
+
+  settings.source = [src1, src2]
+
+For a full list of all classes related to statistical distributions, see
+:ref:`pythonapi_stats`.
+
+
+---------------
+Shannon Entropy
+---------------
+
+To assess convergence of the source distribution, the scalar Shannon entropy
+metric is often used in Monte Carlo codes. OpenMC also allows you to calculate
+Shannon entropy at each generation over a specified mesh, created using the
+:class:`openmc.Mesh` class. After instantiating a :class:`Mesh`, you need to
+specify the lower-left coordinates of the mesh (:attr:`Mesh.lower_left`), the
+number of mesh cells in each direction (:attr:`Mesh.dimension`) and either the
+upper-right coordinates of the mesh (:attr:`Mesh.upper_right`) or the width of
+each mesh cell (:attr:`Mesh.width`). Once you have a mesh, simply assign it to
+the :attr:`Settings.entropy_mesh` attribute.
+
+::
+
+   entropy_mesh = openmc.Mesh()
+   entropy_mesh.lower_left = (-50, -50, -25)
+   entropy_mesh.upper_right = (50, 50, 25)
+   entropy_mesh.dimension = (8, 8, 8)
+
+   settings.entropy_mesh = entropy_mesh
+
+If you're unsure of what bounds to use for the entropy mesh, you can try getting
+a bounding box for the entire geometry using the :attr:`Geometry.bounding_box`
```

Comment:
Ditto, I think it is definitely worthwhile to let users know why a bounding box cannot always be found, with a concrete example.